waspc/src/Generator/JsImport.hs

```diff
@@ -0,0 +1,27 @@
+module Generator.JsImport
+  ( getImportDetailsForJsFnImport,
+  )
+where
+
+import Generator.ExternalCodeGenerator.Common (GeneratedExternalCodeDir)
+import StrongPath (Dir, Path, Posix, Rel, (</>))
+import qualified StrongPath as SP
+import qualified Wasp.JsImport
+
+getImportDetailsForJsFnImport ::
```

Comment:
This function already existed, I just extracted it so it can be reused through the codebase.